### PR TITLE
Add Italian Translation

### PR DESCRIPTION
**Pull request by melaniedrevet**
Contribution to the issue #7

### CODE REVIEW

File: src/i18n/it.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      "qZFw": "Yasha",
+      "qap2": "Goblikaze",
+      "sYMV": "Catapulta",
+      "u3vo": "Assassino fantasma",
+      "uuZv": "Combattenti",
+      "uvZH": "Guardia di zucca",
+      "x68d": "Banditi",
+      "xKS4": "Mago del fuoco",
+      "yivK": "Peltasti",
+      "zX4n": "Sacro spadaccino",
```
Comment (antcalvente) [inline src/i18n/it.json:89]: `Spadaccino Sacro`

File: src/i18n/it.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      "5Lj1": "Nave pirata"
+    },
+    "classes": {
+      "dRmi": "Supporto",
+      "8ZOV": "Mago",
+      "Mcn2": "Guerriero",
+      "kroW": "Assassino",
+      "Hei3": "Tiratore scelto",
+      "PA95": "Carro armato"
```
Comment (antcalvente) [inline src/i18n/it.json:98]: `Tank`

File: src/i18n/it.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      "yivK": "Peltasti",
+      "zX4n": "Sacro spadaccino",
+      "5Lj1": "Nave pirata"
+    },
+    "classes": {
+      "dRmi": "Supporto",
+      "8ZOV": "Mago",
+      "Mcn2": "Guerriero",
+      "kroW": "Assassino",
+      "Hei3": "Tiratore scelto",
```
Comment (antcalvente) [inline src/i18n/it.json:97]: `Tiratore`

File: src/i18n/it.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+      "PA95": "Carro armato"
+    },
+    "heroes": {
+      "Xfwz": "Alchimista pazzo - Hohenheim",
```
Comment (antcalvente) [inline src/i18n/it.json:101]: `Alchimista folle - Hohenheim`

File: src/i18n/it.json [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+      "YI3s": "Ingegnere - Planck",
+      "i1hS": "Menestrello - Dante",
+      "DZGh": "Albero della vita - Verde",
```
Comment (antcalvente) [inline src/i18n/it.json:104]: `Albero della vita - Green`

File: src/i18n/it.json [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+      "WiY9": "Cacciatore della giungla - Diana",
```
Comment (antcalvente) [inline src/i18n/it.json:105]: `Cacciatrice della Giungla - Diana`

File: src/i18n/it.json [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+      "j1ym": "Frost Baron - Kelvin",
```
Comment (antcalvente) [inline src/i18n/it.json:106]: `Barone di Guiaccio - Kelvins`

File: src/i18n/it.json [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+      "OupS": "Stregone Elementale - Illum",
+      "fvns": "Arconte - Moshe",
+      "9bv9": "Barba Nera - Edoardo",
```
Comment (antcalvente) [inline src/i18n/it.json:109]: `Barbanera - Edward`

File: src/i18n/it.json [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+      "nCuH": "Guardiano di Natale - Kriss",
```
Comment (antcalvente) [inline src/i18n/it.json:110]: `Gigante del Natale - Kriss`

File: src/i18n/it.json [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+      "LHtK": "Signore dell'inverno - Ainz",
+      "5OcS": "Il principe dei ladri - Robin Hood",
```
Comment (antcalvente) [inline src/i18n/it.json:112]: `Principe dei Ladri - Robin Hood`

File: src/i18n/it.json [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+      "OQli": "Re Demone Toro - Minotauro",
```
Comment (antcalvente) [inline src/i18n/it.json:113]: `Re Toro Demoniaco - Minotauro`

File: src/i18n/it.json [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+      "Zhfz": "Maestro dei Tarocchi - Cassandra",
```
Comment (antcalvente) [inline src/i18n/it.json:114]: `Maestra dei Tarocchi - Cassandra`

File: src/i18n/it.json [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+      "CbHd": "Insanguinato lupo mannaro - Miller",
```
Comment (antcalvente) [inline src/i18n/it.json:115]: `Lupo Mannaro Succhiasangue - Miller`

File: src/i18n/it.json [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+      "GAcg": "Sakura Blade - Genichiro",
```
Comment (antcalvente) [inline src/i18n/it.json:116]: `Sakura Kiru - Genichiro`

File: src/i18n/it.json [hunk 15]
```diff
@@ -0,0 +1,171 @@
...
+      "ArWp": "Ammiraglio della Nuova Rotta - Drake",
+      "ARHK": "Re del Tuono - Ivan",
```
Comment (antcalvente) [inline src/i18n/it.json:118]: `Re dei Fuilmini - Ivan`

File: src/i18n/it.json [hunk 16]
```diff
@@ -0,0 +1,171 @@
...
+      "2ZWr": "Orologio - Harrison",
```
Comment (antcalvente) [inline src/i18n/it.json:119]: `Orologiaio - Harrison`

File: src/i18n/it.json [hunk 17]
```diff
@@ -0,0 +1,171 @@
...
+      "cBFI": "Aureola splendente - Apollo",
+      "v9oW": "Guardiano dell'Oceano Smeraldo - Poseidone",
```
Comment (antcalvente) [inline src/i18n/it.json:121]: `Re dell'Oceano - Poseidone`

File: src/i18n/it.json [hunk 18]
```diff
@@ -0,0 +1,171 @@
...
+      "DUBg": "Dea della luna - Selene",
+      "qe78": "Il Re Scimmia - Wukong",
```
Comment (antcalvente) [inline src/i18n/it.json:123]: `Re delle Scimmie - Wukong`

File: src/i18n/it.json [hunk 19]
```diff
@@ -0,0 +1,171 @@
...
+      "tPKl": "Consiglio ardente - Davison",
```
Comment (antcalvente) [inline src/i18n/it.json:124]: `Emissario del Drago - Davison`

File: src/i18n/it.json [hunk 20]
```diff
@@ -0,0 +1,171 @@
...
+      "4ZBz": "Duca Scarlatto - Dracula",
```
Comment (antcalvente) [inline src/i18n/it.json:125]: `Conte Vampiro - Dracula`

File: src/i18n/it.json [hunk 21]
```diff
@@ -0,0 +1,171 @@
...
+      "SaVx": "Royal Christmas Ball - Chione",
```
Comment (antcalvente) [inline src/i18n/it.json:126]: `Regina della Terra Ghiacciata - Chione`

File: src/i18n/it.json [hunk 22]
```diff
@@ -0,0 +1,171 @@
...
+      "yW9G": "Signore dell'odio - Mefisto",
```
Comment (antcalvente) [inline src/i18n/it.json:127]: `Signore dell'Odio - Mephisto`

File: src/i18n/it.json [hunk 23]
```diff
@@ -0,0 +1,171 @@
...
+      "7FYw": "Horus il Vecchio - Horus",
```
Comment (antcalvente) [inline src/i18n/it.json:128]: `Horus l'Anziano - Horus`

File: src/i18n/it.json [hunk 24]
```diff
@@ -0,0 +1,171 @@
...
+      "OJ6m": "Il re di Camelot - Artù",
```
Comment (antcalvente) [inline src/i18n/it.json:129]: `Re di Camelot - Arthur`

File: src/i18n/it.json [hunk 25]
```diff
@@ -0,0 +1,171 @@
...
+      "NUMk": "Re intrepido - Beowulf",
```
Comment (antcalvente) [inline src/i18n/it.json:130]: `Intrepido Re - Beowulf`

File: src/i18n/it.json [hunk 26]
```diff
@@ -0,0 +1,171 @@
...
+      "MXhs": "La volpe a nove code - Aly",
+      "U5Bu": "Tentacoli degli abissi - Kraken",
+      "MVQQ": "Grim Reaper - Azrael",
```
Comment (antcalvente) [inline src/i18n/it.json:133]: `Tetro Mietitore - Azrael`

File: src/i18n/it.json [hunk 27]
```diff
@@ -0,0 +1,171 @@
...
+      "uLhj": "Re degli dei - Zeus",
+      "9VqF": "Lancia ardente - Nezha",
```
Comment (antcalvente) [inline src/i18n/it.json:135]: `Lancia Infuocata - Nezha`

File: src/i18n/it.json [hunk 28]
```diff
@@ -0,0 +1,171 @@
...
+      "3PGT": "Dea della guerra - Atena",
+      "d6Wc": "Padre degli dei - Odino",
+      "AgM6": "Lo spirito della lampada - Jinn",
```
Comment (antcalvente) [inline src/i18n/it.json:138]: `Spirito della lampada - Jinn`

File: src/i18n/it.json [hunk 29]
```diff
@@ -0,0 +1,171 @@
...
+  "gameData": {
+    "grades": {
+      "XyDE": "Comune",
+      "3XE9": "Raro",
+      "fod3": "Epico",
+      "BB4h": "Leggendario"
+    },
+    "races": {
+      "HVSp": "Umano",
+      "u2TP": "selvaggio",
```
Comment (antcalvente) [inline src/i18n/it.json:40]: `Orco`

File: src/i18n/it.json [hunk 30]
```diff
@@ -0,0 +1,171 @@
...
+      "z18Y": "Sacro",
+      "T1WD": "Buio"
```
Comment (antcalvente) [inline src/i18n/it.json:42]: `Oscuro`

**Review by antcalvente — CHANGES_REQUESTED** [review]
Hello! 

Thanks a lot for adding the translations for Italian, I've just put some comments for the troops names that don't match the in game name. 
It's important to keep the same name as otherwise it would be really difficult for users to find their troops as it's order alphabetically and they'll expect to find the same exact name

Once fixed we'll merge the changes and activate Italian on the site :)

**Review by melaniedrevet — COMMENTED** [review]
Hello !

Thank you for your feedback, I corrected the troop names as requested ! Indeed, it will be more consistent for users !

**Review by antcalvente — APPROVED** [review]
Thanks for the quick fixes! 

We'll merge it as soon as possible, add you in the list of contributors and then enable the Italian language in the UI

**Review by Jonur — APPROVED** [review]

**Comment by Jonur** [comment]
Thank you both @melaniedrevet and @antcalvente for this! <3 <3 <3 PR approved and @melaniedrevet I am adding you to the contributors.

**Pull request merged**